draft-ietf-taps-transport-security.md

```diff
@@ -240,8 +240,9 @@ peer to the other. This data may contain handshake messages or raw application d
 TLS is the composition of a handshake and record protocol {{I-D.ietf-tls-tls13}}.
 The record protocol is designed to marshal an arbitrary, in-order stream of bytes from one endpoint to the other.
 It handles segmenting, compressing (when enabled), and encrypting data into discrete records. When configured
-to use an AEAD algorithm, it also handles nonce generation and encoding for each record. The record protocol is
-hidden from the client behind a byte stream-oriented API.
+to use an authenticated encryption with associated data (AEAD) algorithm, it also handles nonce 
+generation and encoding for each record. The record protocol is hidden from the client behind a 
+byte stream-oriented API.
```

Comment:
"byte stream-oriented" sounds weird, because it associates stream with oriented more than byte.

"bytestream-oriented"?
"byte-stream oriented"?

Reply:
Either or works for me.